app/graphql/mutations/geo.rb

```diff
@@ -0,0 +1,21 @@
+module Geo
+  def self.locate(ip_address)
+    return {} unless ip_address
+
+    result = Geocoder.search(ip_address)
+
+    if match = result[0]
+      {
+        country_name: match.country,
+        country_code: match.country_code,
+        city_name: match.city,
+        latitude: match.latitude,
+        longitude: match.longitude
+      }
+    else
+      {}
+    end
+  rescue StandardError
```

Comment:
this will swallow all the normal errors from the above code, normally you would set which errors you'd like to "control" and let unexpected ones raise so you can fix them...avoids the issue of the geocoder silently failing and we don't notice. 

I think this was copied so no need to correct it, just mentioning it.